charts/gatus/README.md

```diff
@@ -61,6 +61,7 @@ _See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command doc
 | `ingress.hosts`                           | Ingress accepted hostnames                      | `["chart-example.local"]`           |
 | `ingress.tls`                             | Ingress TLS configuration                       | `[]`                                |
 | `env`                                     | Extra environment variables passed to pods      | `{}`                                |
+| `envFrom`                                 | Extra environment variables passed to pods      | `[]`                                |
```

Comment:
Could you update the `envFrom` description you added in the docs? `Extra environment variables passed to pods` for `envFrom` isn't really accurate.

Something like `Additional envFrom configuration to use a Secret or a ConfigMap in an environment variable` would make more sense

Reply:
It's done :+1: